src/rev_ai/models/basejob.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""Base Job model"""
+
+from .asynchronous.job_status import JobStatus
+
+
+class BaseJob:
```

Comment:
I created this class for insights job and async job to inherit but there doesn't seem to be any actual benefit to this. I'll probably remove it